texttospeech/api/README

```diff
@@ -0,0 +1,59 @@
+# .NET Cloud TextToSpeech API Samples
+
+A collection of samples that demonstrate how to call the 
+[Google Cloud TextToSpeech API](https://cloud.google.com/text-to-speech/) from C#.
+
+This sample requires [.NET Core 2.0](
+    https://www.microsoft.com/net/core) or later.  That means using
+[Visual Studio 2017](
+
```

Comment:
This Markdown link looks broken? it falls strangely onto a newline

Reply:
Done